automerge-backend-wasm/scripts/js_tests.sh

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env bash
+
+if [[ -e "../../automerge" ]]
```

Comment:
This seems to hardcode the location of the `automerge` checkout. Can we instead clone it into the working directory and then add to `.gitignore`?

Reply:
This is set-up so that given a `automerge-rs` folder, this will checkout `automerge` as a folder underneath the same parent directory so there's no absolute paths (if that's what you mean by hardcode).

I'm not sure how we can avoid hard coding. Won't we always have to clone the `automerge` repository to some specified location?

Reply:
Yeah what I'm suggesting is that we should clone it to a subdirectory of the `automerge-rs` repo, then add whatever directory we clone it to the `.gitignore`. I _think_ that should do the trick.

IMO We definitely shouldn't clone to somewhere outside the project directory. I think that's quite surprising behaviour that will annoy a lot of people and break build systems.

Reply:
I agree that we should also just have it inside the automerge-rs directory somewhere. I wouldn't want it interfering externally.